docs/source/Quickstart/03_psi.md

```diff
@@ -55,13 +62,13 @@ ssh cloud-user@<obtained IP> -i ~/.ssh/rhos-jenkins
 ```
 
 5. **Install ci-framework dependencies**
+
 ```
 sudo yum -y groupinstall "Development Tools"
 sudo yum -y install  qemu-img qemu-kvm libguestfs libguestfs-tools-c wget pip
 python3 -m pip install --user ansible jmespath beautifulsoup4
-
 ```
 
 6. **Continue with Quickstart**
 
-Continue with ci-framework deployment as described by the [Quickstart section](https://github.com/openstack-k8s-operators/ci-framework/wiki/Quickstart).
+Continue with ci-framework deployment as described by the [Virtualized environment setup section](https://ci-framework.readthedocs.io/en/latest/Quickstart/02_nested_virt.html).
```

Comment:
Maybe a good opportunity to use https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role ? Though I'm not 100% sure it will work with markdown engine.

Reply:
markdown does not directly support reST as far as I know. At least not directly.